doc/mgmt/Show_Running_Configuration_HLD.md

```diff
@@ -0,0 +1,495 @@
+
```

Comment:
I'm going to focus my review on requirements - I don't understand the rest of it well enough to add any value there.